examples/scripts/iframe/index.mjs

```diff
@@ -0,0 +1,68 @@
+import fs from 'fs';
+import Babel from '@babel/standalone';
+import Handlebars from 'handlebars';
+import formatters from '../../src/app/helpers/formatters.mjs';
+import { dirname } from 'path';
+import { fileURLToPath } from 'url';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+
+const MAIN_DIR = `${__dirname}/../../`;
+
+const EXAMPLE_CONSTS = [
+    "vshader",
+    "fshader",
+    "fshaderFeedback",
+    "fshaderCloud",
+    "vshaderFeedback",
+    "vshaderCloud"
+];
+
+function retrieveConstString(data, name) {
+    const start = data.indexOf(`const ${name} = `);
+    if (start < 1) return;
```

Comment:
The `const vshaderCloud = ...` is technically allowed to be the first line, possibly resulting in 0. Only -1 is "not found".
```suggestion
    if (start < 0) return;
```

Reply:
Done, thanks! 👍